python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -81,16 +86,16 @@ def _execUpdateAction(self):
             if self.should_stop():
                 break
 
-            tpFreeThreads -= 1
             #setattr(threading.currentThread(), 'action', action)
             log.debug("Qin's size is currently: %d" % Qin.qsize())
             log.debug("%s running..." % threading.currentThread())
-
+            self._alive = True
```

Comment:
How does this differ from [`is_alive()`](https://docs.python.org/2.6/library/threading.html#threading.Thread.is_alive)? It seems like a potentially confusing naming.